ubo_app/services/020-keyboard/shortcuts.md

```diff
@@ -0,0 +1,68 @@
+# Keyboard & Keypad Shortcuts Guide
+
+## Basic Navigation
+| Shortcut | Alternative Keys | Action |
+|----------|-----------------|---------|
+| UP | K | Move selection up |
+| DOWN | J | Move selection down |
+| BACK | ESC, H, ← | Go back one level |
+| HOME | Backspace | Return to home menu |
+| L1 | 1 | Select first item |
+| L2 | 2 | Select second item |
+| L3 | 3 | Select third item |
+
+## Special Commands
+| Shortcut | Action |
+|----------|---------|
+| HOME + L1 | Take screenshot |
+| HOME + L2 | Take snapshot |
+| HOME + L3 | Start/Stop recording sequence |
+| BACK + L3 | Replay recorded sequence |
+| HOME + BACK | Exit application |
+| CTRL + M | Toggle input mute |
```

Comment:
We don’t have `CTRL + M` and this list is not listing keyboard shortcuts
I think we can remove this row